changelog/fragments/pip3-21.1.yaml

```diff
@@ -0,0 +1,4 @@
+entries:
+  - description: >
+      Pinned pip3 to 21.1 in the ansible-operator image to fix https://github.com/pypa/pip/pull/9827
```

Comment:
This is only kinda user-facing because they may install other python packages using the pre-installed pip3 in their operator's image, i.e. this is only a build vulnerability that most image consumers won't be exposed to.